src/lib/utils/index.ts

```diff
@@ -16,8 +14,6 @@ import intersection from 'lodash/intersection'
 import isUndefined from 'lodash/isUndefined'
 import union from 'lodash/union'
 
-export * as evm from './evm'
```

Comment:
in my opinion this is super annoying and we should refrain from doing it

Reply:
Yeah totally agree with that, finding it hard to navigate the `export * as foo` exports as well

Reply:
I have 0 objection to discontinuing this pattern.

lol at "import zoo".